src/Orchard.Web/Modules/Orchard.ContentPreview/Drivers/ContentPreviewDriver.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Orchard.ContentManagement.Drivers;
+using Orchard.Core.Common.Models;
+
+namespace Orchard.ContentPreview.Drivers {
+    public class ContentPreviewDriver : ContentPartDriver<CommonPart> {
```

Comment:
Nem kellene, hogy csak a CommonParttal működjön. Egy közvetlen IContentPartDriver implementációból, esetleg valahogy handlerből a BuildEditor eventnél hozzáadhatod közvetlenül.

Reply:
Ez így ördögtől való? Holnap még gondolkozom, de látszólag működik.
```
        protected override void BuildEditorShape(BuildEditorContext context) {
            context.Shape.Zones["Sidebar"].Add(_shapeFactory.ContentPreview_Button(Model: context.ContentItem));
        }
```

Reply:
Ez eléggé. Az úgy jó, hogy a placementtel lehet elhelyezni. Amúgy nézd meg, hogy pl. a Save meg Publish button hogy kerül ki, asszem azok is machinálhatók placementben, pedig nincs hozzájuk külön content part.

Reply:
Basszus, pedig még ez nekem is eszembe jutott anno, de csak a PublishLatert néztem meg az pedig külön viewmodellel baszkódás, saját ContentParttal stb...